### DISCUSSION TITLE
Cross-references for tables and docx

### DISCUSSION THREAD

**Discussion by lorenzoFabbri** [quarto-dev/quarto-cli#6816] [Q&A]
Just to be 100% sure: there is currently absolutely no way of using packages like `gt` and `flextable`, and get cross-references to work with docx, right?

@cscheid @davidgohel @rich-iannone @cderv

**Comment by davidgohel** [comment]
(I am not part of Quarto team @lorenzoFabbri). flextable captions and cross ref works with quarto 1.3 for HTML and PDF by doing nothing about it (captions and cross ref are managed by Quarto). For Word, I understood it was planned for Quarto 1.4.

**Reply by lorenzoFabbri** [reply]
Okay thanks.

**Comment by cderv** [comment]
There is a big work ongoing with crossref to make them better and cover a lot of things. You can follow this: 
* https://github.com/quarto-dev/quarto-cli/issues/4944

Two main issues regarding this discussion are listed in the work covered 
* https://github.com/quarto-dev/quarto-cli/issues/3600
* https://github.com/quarto-dev/quarto-cli/issues/3601

It will also surely requires adaptation in what is produced by those packages. Some issues to follow 

* https://github.com/rstudio/gt/issues/1140
* https://github.com/davidgohel/flextable/issues/494

Those issue will be updated once something is ready. 

Patience is key here.

**Reply by lorenzoFabbri** [reply]
Suppose I forget about these packages and manually format the tables, and that I just need cross-referencing in docx, is there a way to achieve this? Thanks.

**Reply by cderv** [reply]
If you use Markdown tables tables as outputs, then you can use the reference syntax  (https://quarto.org/docs/authoring/tables.html#cross-references)

You could also target HTML output (https://quarto.org/docs/authoring/tables.html#html-tables)  as parsed them as Markdown inside Quarto. 

Tools that produce openxml content for docx are not parsed - hence the specific treatment that should happen for cross ref to work, and this is part of #4944 big evolution
